integrationtests/cdi-weld-se-it/src/test/java/org/infinispan/integrationtests/cdi/weld/WeldStandaloneTest.java

```diff
@@ -25,6 +25,9 @@ public void testWeldStandaloneInitialisation() throws Exception {
 
       //then
       assertEquals(retrievedValue, "abcd");
+
+      //cleanup
+      weld.shutdown();
```

Comment:
In this particular test - no... Weld adds a shutdown hook. But let me add if for readability reasons